src/controllers/enter.information.controller.ts

```diff
@@ -1,20 +1,19 @@
 import { Session } from "ch-node-session-handler";
 import { NextFunction, Request, Response } from "express";
 import ObjectionCompanyProfile from "model/objection.company.profile";
-import { SESSION_OBJECTION_ID } from "../constants";
-import { OBJECTIONS_DOCUMENT_UPLOAD } from "../model/page.urls";
+import { CHANGE_ANSWER_KEY, SESSION_OBJECTION_ID } from "../constants";
+import { OBJECTIONS_CHECK_YOUR_ANSWERS, OBJECTIONS_DOCUMENT_UPLOAD } from "../model/page.urls";
 import { Templates } from "../model/template.paths";
-import { updateObjectionReason } from "../services/objection.service";
+import { Objection } from "../modules/sdk/objections";
+import { updateObjectionReason, getObjection } from "../services/objection.service";
 import {
   retrieveAccessTokenFromSession,
   retrieveCompanyProfileFromObjectionSession,
   retrieveFromObjectionSession,
 } from "../services/objection.session.service";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
-  return res.render(Templates.ENTER_INFORMATION, {
-    templateName: Templates.ENTER_INFORMATION,
-  });
+  await renderPageWithSessionDataIfPresent(req, res, next)
```

Comment:
I'm not sure if putting this functionality in another method does much, it may be easier to read if the code is just in the get method.